network/src/commonMain/kotlin/com/wire/kalium/network/api/base/authenticated/notification/EventContentDTO.kt

```diff
@@ -294,7 +299,60 @@ sealed class EventContentDTO {
         ) : FeatureConfig()
     }
 
+    @Serializable
+    sealed class UserProperty : EventContentDTO() {
+        @Serializable
+        @SerialName("user.properties-set")
+        data class PropertiesSetDTO(
+            @SerialName("key") val key: String,
+            @SerialName("value") val value: FieldKeyValue,
+        ) : UserProperty()
+
+        @Serializable
+        @SerialName("user.properties-delete")
+        data class PropertiesDeleteDTO(
+            @SerialName("key") val key: String,
+        ) : UserProperty()
+
+        enum class PropertyKey(val key: String) {
```

Comment:
i guess this enum is not used?

Reply:
Is it used, to check for keys on event handling and now for API calls, so I moved it now to the API classes for better understanding 👍